### PR TITLE
[Feature] - Added Avatar component in the library

### PR DESCRIPTION
**Pull request by parthpandyappp**
### Directory structure and file justification
Added a new directory named alerts under which markup has been written in `avatars.html` and the stylesheet file is written in `avatars.css`. The stylesheet file `avatars.css` only consists of the styles related to buttons i.e it is a feature-specific stylesheet file that is later getting integrated with the main stylesheet file in the root directory i.e `styles.css`

### Explicit integration of stylesheet file
As we see, many open-source CSS libraries generally have a CDN link that is embedded and linked to the head section of any markup file. Similarly, a single main styles.css file is responsible to provide access to all the other stylesheets.

## About this PR
This PR adds a Avatar component fo different types

### CODE REVIEW

File: avatars/avatars.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            <img
+              class="avatar avatar-sm square"
+              src="https://randomuser.me/api/portraits/men/20.jpg"
+              alt=""
+            />
+          </div>
+
+          <li>Textual Avatars</li>
+          <div class="disp-avatar">
+            <p class="avatar avatar-xl round txt-avatar">P</p>
```
Comment (TejasShekar) [inline avatars/avatars.html]: The properties of avatar are overwritten by avatar-xl. They both have same props. Use only one of them. Do the same for other elements as well.

Also, in mobile screen, the text P is oval and not round. Check that too.

**Review by TejasShekar — COMMENTED** [review]
Looks good for me. Have a look at the changes I have mentioned.

File: avatars/avatars.html (hunk 1 above)
Comment (parthpandyappp) [reply avatars/avatars.html]: Sure, I'll update it now.

**Review by Logan1x — APPROVED** [review]
LGTM

**Pull request merged**